lib/legacy_client/client.rb

```diff
@@ -97,6 +97,8 @@ def request(method, path, payload = nil, headers = {})
         headers[:content_type] = "application/json"
       end
 
+      headers = headers.merge(GdsApi::GovukHeaders.headers)
```

Comment:
https://github.com/alphagov/rummager/blob/23e84e8d4e6240254b56d220f22443b17e49fb92/lib/legacy_client/client.rb#L108 seems to suggest `headers` can be nil. Can we remove that line?